lib/route/addr.c

```diff
@@ -840,6 +851,25 @@ int rtnl_addr_delete(struct nl_sock *sk, struct rtnl_addr *addr, int flags)
  * @{
  */
 
+int rtnl_addr_set_alias(struct rtnl_addr *addr, const char *alias)
+{
+	if (strlen(alias) > sizeof(addr->a_alias) - 1)
```

Comment:
since the getter can return `%NULL`, possibly the setter also should accept %NULL, to make the value unset. Yes, that is not done for other attributes, which I find unfortunate :(